SimPEG/Mesh/InnerProducts.py

```diff
@@ -1,5 +1,7 @@
 from scipy import sparse as sp
-from SimPEG.Utils import *
+from SimPEG.Utils import (sub2ind, sdiag, invPropertyTensor, TensorType,
```

Comment:
this starts to get a bit out of hand... should we just `import Utils` and do `Utils.XXX` ?